code/__HELPERS/sanitize_values.dm

```diff
@@ -74,7 +74,7 @@
 	return crunch + .
 
 /proc/sanitize_ooccolor(color)
-	var/list/HSL = rgb2hsl(hex2num(copytext(color,2,4)),hex2num(copytext(color,4,6)),hex2num(copytext(color,6,8)))
+	var/list/HSL = rgb2hsl(hex2num(copytext_char(color, 2, 4)), hex2num(copytext_char(color, 4, 6)), hex2num(copytext_char(color, 6, 8)))

```

Comment:
Generally i feel all these color helpers would be better off with `if(length != length_char) CRASH` somewhere.

Reply:
I tried to see if there were any special Unicode symbols for hex digits and there does not appear to be any, which makes sense. I'll also assume # is always gonna be the opener. Thus yeah I agree with you.